core/src/test/java/org/elasticsearch/search/aggregations/InternalAggregationTestCase.java

```diff
@@ -122,4 +135,56 @@ protected final T createUnmappedInstance(String name) {
     protected NamedWriteableRegistry getNamedWriteableRegistry() {
         return namedWriteableRegistry;
     }
+
+    protected ContextParser<Object, Aggregation> instanceParser() {
+        throw new UnsupportedOperationException("Not implemented yet");
+    }
```

Comment:
Instead of providing the instance parser by overwriting this method in the subtests, can subtest simply provide their own xContentRegistry with the appropriate parser by overwriting `getNamedWriteableRegistry()` from AbstractWireSerializingTestCase? I might be missing something though.